File: app/src/main/java/com/smoothradio/radio/core/data/local/AppDatabase.java
package com.smoothradio.radio.core.data.local;

import androidx.room.Database;
import androidx.room.RoomDatabase;

import com.smoothradio.radio.core.model.RadioStation;

@Database(entities = {RadioStation.class}, version = 1)
public abstract class AppDatabase extends RoomDatabase {
    public abstract RadioStationDao radioStationDao();
}

File: app/src/main/kotlin/com/smoothradio/radio/core/data/local/AppDatabase.kt
package com.smoothradio.radio.core.data.local

import androidx.room.Database
import androidx.room.RoomDatabase
import com.smoothradio.radio.core.data.local.model.RadioStationEntity

@Database(entities = [RadioStationEntity::class], version = 2)
abstract class AppDatabase : RoomDatabase() {
    abstract fun radioStationDao(): RadioStationDao
}
